### DISCUSSION TITLE
Mode vacation

### DISCUSSION THREAD

**Discussion by RSickenberg** [community/community#17599] [Mobile]
Hello,

As a developer, and in vacation, I would love to have a vacation mode like other tools to turn off notifications until à given date. 

Bonus point if the push notification isn't sent so my device don't have to awake the process 🙂. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.67.0 (68716861)
Device: iPhone 13 Pro Max (iOS 15.5)
</details>

**Comment by HenSquared** [comment]
Hi @RSickenberg, thanks a bunch for this feature request. I've opened an issue to track this and will discuss further internally with the team. 

As a workaround for this, maybe you can disable GitHub notifications manually, then reenable them once you would like to receive them again after vacation. 


[Change notification settings on iPhone
](https://support.apple.com/guide/iphone/change-notification-settings-iph7c3d96bab/ios#:~:text=To%20turn%20off%20notifications%20selectively,then%20turn%20off%20any%20app.)
> To turn off notifications selectively for apps, go to Settings > Notifications > Siri Suggestions, then turn off any app.

**Reply by RSickenberg** [reply]
Yes, I did this as a workaround but I still want this feature :)

**Reply by wlandau** [reply]
I would love to use built-in temporary interaction limits for vacations and weekends (like this one) so that requests (like this one) do not pile up while I am away. Just a couple more things I would need:

1. The ability to set a fully custom expiration date/time on interaction limits.
2. A conspicuous customizable banner at the top of the repo page when these interaction limits are active.

I have seen several attempts at workarounds in GitHub Actions, but the ones I have tried don't work, and they don't have obvious ways of informing users. Even if they did work, it is burdensome to test an action when I am trying to unplug.

**Comment by danp** [comment]
Related, I think:

I wish I could put an org I'm a member of into vacation or similar mode. For example, while I'm on break from work I might still use GitHub for personal/etc things but I don't want to see work-related stuff in my notifications or anything.

I guess I could leave the org but that seems like a bit much.

(Oh, darn, just realized this is under "Mobile." I had searched for "vacation" and landed here. What I would like extends beyond mobile.)

**Reply by Sergi0Olmedo** [reply]
Creo que podrías separa 2 cuentas una personal y una de trabajo
